corebehrt/functional/data_check.py

```diff
@@ -19,3 +21,17 @@ def log_features_in_sequence(data: dict) -> None:
     logger.info("Example features: ")
     for k, v in data.features.items():
         logger.info(f"{k}: {v[0]}")
+
+
+def check_patient_counts(
```

Comment:
Should this only issue a warning or is this an error?

Reply:
I think this method is no longer needed. Also, we'll get rid of the ConceptLoaderLarge.
It's probably a good idea to tackle create_outcomes next to rewrite with dask

Reply:
Adressed in a8129132b34860ad4e39afbd1131375127d61d3f